### ISSUE TITLE
Debian repo

### ISSUE THREAD

**Issue by n1-cz** [wagoodman/dive#62]
Hi there, any plans with pushing this awesome piece of software into Debian repo? Or running your own?

**Comment by wagoodman** [comment]
Added an issue in goreleaser (https://github.com/goreleaser/goreleaser/issues/864) to add SRPM support.

**Comment by rzr** [comment]
It could make sense to also send a RFP to debian tracker, but package might be renamed because it would conflict with https://bugs.debian.org/726779 ? maybe dive-docker or go-dive ? any preference ?

**Comment by wagoodman** [comment]
Good catch, @rzr . In the future I'm hoping to integrate more tools and engines with dive, so tying it directly with `docker` would be misleading in the future. The fact it's go is fairly arbitrary too.  How about `dive-container`? I'm open to more suggestions.

**Comment by KOLANICH** [comment]
GitLab CI allows to create GitLab Pages with pipeline execution. This allows creation of a debian repo. It can be a temporary solution for the time it is not in the official repos.

**Comment by limakzi** [comment]
@KOLANICH How about running your own gitlab-runner? (On AWS, for example) 
Keep the GPG-key there. In ASM. In Vault. Anywhere.  
Store password to this key as a secret-variable?

More-over, use intermediate GPG key. And never GPG-CA anywhere?

**Comment by KOLANICH** [comment]
> How about running your own gitlab-runner? (On AWS, for example)

Only on dedicated physical machine. Remember about microarchitectural cross-vm attacks.

> Keep the GPG-key there. In ASM. In Vault. Anywhere.

Probably in a HSM.

Though all these measures are infeasible. Here is a feasible measure that should be done anyway:

* separate builds to the ones done and signed manually and ones done automatically and say that ones done automatically are less trusted;

* make the builds reproducible to allow everyone to check if build infrastructure acts trustworthily.

**Comment by alexanderadam** [comment]
Just to be sure: is there any official RFP for Debian somewhere? Or something similar that can be tracked?

**Comment by rzr** [comment]
None I know but I can create one for dive-container ? or can you suggest an other name ?

**Comment by theAkito** [comment]
> 
> 
> None I know but I can create one for dive-container ? or can you suggest an other name ?

You should just call it `dive`. There is currently no package with that name, so you can use this one. Suffixes are usually only applied for a variation of software or if the name is already taken. If it is not, just use the original name, which is `dive`.

**Comment by tonymet** [comment]
what happened with the deb project?

**Comment by tonymet** [comment]
I submitted a "request for package" seeking a sponsor (uploader) for the package.  If I find one, I can help with the packaging and prep in the next release.  Please comment here if you can help with the packaging and testing efforts

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1093710

**Comment by rzr** [comment]
good start may i suggest you to start working somewhere (ie in a fork) once you have something you can join an existing debian team for cooperative maintenance.

**Comment by tonymet** [comment]
> good start may i suggest you to start working somewhere (ie in a fork) once you have something you can join an existing debian team for cooperative maintenance.

can you help out with building the package or testing? i'm looking for volunteers.  I can get the rules and watch files  . I may  need help running tests, doing install, adding some tests for debian package
